pkgs/tools/X11/setroot/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, lib, fetchFromGitHub, libX11, imlib2
+, enableXinerama ? true, libXinerama ? null
+}:
+
+assert enableXinerama -> libXinerama != null;
+
+let version = "1.4.4"; in
+
+stdenv.mkDerivation {
+  name = "setroot-${version}";
+
+  src = fetchFromGitHub {
+    owner = "ttzhou";
+    repo = "setroot";
+    rev = "v${version}";
+    sha256 = "0vphma0as8pnqrakdw6gaiiz7xawb4y72sc9dna755kkclgbyl8m";
+  };
+
+  buildInputs = [ libX11 imlib2 ]
+    ++ stdenv.lib.optional enableXinerama libXinerama;
+
+  buildFlags = if enableXinerama then "xinerama=1" else "xinerama=0";
+
+  installPhase = ''
+    make PREFIX="" DESTDIR="$out" install
```

Comment:
Use `installFlags` to set PRFIX and DESTDIR instead of overwriting `intallPhase`.